.github/workflows/helm-validation.yml

```diff
@@ -0,0 +1,407 @@
+name: Test Chart
+
+on:
+  push:
+
+permissions:
+  id-token: write
+  contents: write  # Grant write access to the repository contents
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+           - mode: Sensor
+           - mode: AutoScaler
+
+    name: Test (${{ matrix.mode }})
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4.3.1
+
+      - name: Add required Helm repos
+        run: |
+          helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
+          helm repo update
+
+      - uses: actions/setup-python@v5.3.0
+        with:
+          python-version: '3.13.x'
+          check-latest: true
+
+      - name: Set up chart-testing
+        uses: helm/chart-testing-action@v2.7.0
+
+      - name: Run chart-testing (lint)
+        run: ct lint --charts . --config .ct.yaml --target-branch ${{ github.event.repository.default_branch }}
+
+      - name: Detect related PR
+        id: detect_pr
+        env:
+          GH_TOKEN: ${{ github.token }}
+          REPO: ${{ github.repository }}
+          BRANCH: ${{ github.ref_name }}
+        run: |
+          set -e
+          owner_repo="$REPO"
+          # List open PRs whose head matches owner:branch
+            json=$(curl -s -H "Authorization: Bearer $GH_TOKEN" \
+              "https://api.github.com/repos/$owner_repo/pulls?state=open&head=${owner_repo%%/*}:$BRANCH")
+          count=$(echo "$json" | jq 'length')
+          if [ "$count" -gt 0 ]; then
+            # take first
+            draft=$(echo "$json" | jq '.[0].draft')
+            number=$(echo "$json" | jq '.[0].number')
+            echo "Found open PR #$number (draft=$draft)"
+            echo "has_pr=true" >> "$GITHUB_OUTPUT"
+          else
+            echo "No open PR for branch $BRANCH"
+            echo "has_pr=false" >> "$GITHUB_OUTPUT"
+          fi
+
+      - name: Create kind cluster
+        if: ${{ steps.detect_pr.outputs.has_pr == 'true' }}
+        uses: helm/kind-action@v1.12.0
+
+      - name: Apply fake EBS CSI resources
```

Comment:
if your applying a "fake" one, why the need for the 
```
helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
```
above? or why not just apply the the csi driver from there?

Reply:
@BorisKofman you do resolve without changing the code nor answering my comment